### PR TITLE
[StructuralMechanics] fixing local-axis plotting for beams & shells

### PR DESCRIPTION
**Pull request by philbucher**
This PR fixes the local-axis plotting for beams and shells
Broken for some time already since sth in the way the Gauss-Point results are written has changed => I think it was broken in #1257

@KlausBSautter for some reason writting results did not work for me, we should have a look later

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp [hunk 1]
```diff
@@ -472,75 +472,6 @@ void ShellThickElement3D3N::CalculateOnIntegrationPoints(const Variable<double>&
...
-        // Compute the local coordinate system.
-        ShellT3_LocalCoordinateSystem localCoordinateSystem(mpCoordinateTransformation->CreateReferenceCoordinateSystem());
-
-        // Get local axis 1 in flattened LCS space
-        Vector3 localAxis1 = localCoordinateSystem.P2() - localCoordinateSystem.P1();
-
-        // Perform rotation of local axis 1 to fiber1 in flattened LCS space
-        Matrix localToFiberRotation = Matrix(3, 3, 0.0);
-        double fiberSectionRotation = mSections[0]->GetOrientationAngle();
-        double c = std::cos(fiberSectionRotation);
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp:521]: const, and not very descriptive variables names

**Review by loumalouomega — DISMISSED** [review]
Minor comment from my side

File: applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp (hunk 1 above)
Comment (philbucher) [reply applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp:521]: I am removing this ...

**Comment by KlausBSautter** [comment]
@philbucher I fixed the issue with 'bad argument' error msg but there is still sth wrong with:

> Exception: The specified reference file name "/home/klaus/software/Kratos_Main/applications/StructuralMechanicsApplication/tests/local_axis_visualization_ref_result_files/local_axis_CrLinearBeamElement3D2N_0.post.res.ref" is not valid!

**Comment by philbucher** [comment]
@loumalouomega we fixed all problems, can you please have a second look?

**Review by loumalouomega — APPROVED** [review]

**Pull request merged**